src/components/cards/ValidationList/ValidationList.tsx

```diff
@@ -2,25 +2,29 @@ import { useIsMobile } from '../../../hooks/';
 import { ValidationObject } from '../../../types';
 import { CircleCheck, CircleXIcon } from '../../icons';
 
-function ValidationList({ validations }: { validations: ValidationObject[] }) {
+function ValidationList({
+  validations,
+  customStyle,
```

Comment:
we've used a few different prop names for this, let's be consistent.

Reply:
updated